### PR TITLE
Remove usage of index_schema_history instead use pg_indexes

### PR DESCRIPTION
**Pull request by vishalhaven**

### CODE REVIEW

File: components/persistor/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "@haventech/persistor",
     "description": "A subclass of supertype that serializes to and reconstitutes from MongoDB or SQL databases",
     "homepage": "https://github.com/haven-life/persistor",
-    "version": "12.0.0",
```
Comment (kamranHaven) [inline components/persistor/package.json]: Lets point the MR to latest master, which is already published. 
Also this would be a feature MR not a bug fix bump.

File: components/persistor/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+    "version": "12.0.1",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
```
Comment (kamranHaven) [inline components/persistor/package.json:7]: Looks like we have a test case failure (pipeline seems to be failing)

can we make sure to add unit tests around: 

1. no warning message is created if we are deleting an index that is already deleted
2. no warning message is created if we insert an index that was already created
3. If a new index is added in schema file, then pg_indexes has that index.
4. If that same index is deleted from schema file, pg_indexes removes it without impacting any other index
5. Create invalid index where column doesn't exist
6. update index (valid and invalid case)
7. delete index where column is deleted. 


_**Regression:**_
We would also test regression against some large repo like haven and run their startup. 
1. We would need to test above listed scenarios for a blank db (history table empty to mimic fresh db). In a fresh db, history table wont be created. However the pg_indexes initially without the user added indexes, would get populated with the indexes from schema and the ones created by amorphic. 

2. Then we would also want to test with an existing history table (uat / prod scenario) that this feature, stops incrementing the rows in history table, and not delete history table or its contents. However, any adds, updates and deletes to indexes are correctly reflecting.

Comment (vishalhaven) [reply components/persistor/package.json:7]: @kamranHaven  I have covered all the scenarios you mention in the comments. 

About the regression:

1. I have tested with blank db -> works as expected. We need to double the indexes created by the amorphic one. 
2.  Not sure how I should test it. I'll check with the PA repo if there are any changes in the schema/indexes/columns and maybe try to run a startup.
